packages/now-cli/package.json

```diff
@@ -130,8 +127,6 @@
     "get-port": "5.1.1",
     "glob": "7.1.2",
     "http-proxy": "1.17.0",
-    "ignore": "4.0.6",
```

Comment:
I'm surprised, I thought we used `ignore` in v2 also 🤔

Reply:
@styfle yes, but it's in `now-client`

Reply:
I think it's used by `now dev` as well? 🤔